packages/mobile/src/screens/profile-screen/ProfileTabs/RepostsTab.tsx

```diff
@@ -41,6 +62,7 @@ export const RepostsTab = () => {
       lazy={lazy}
       actions={feedActions}
       lineup={lineup}
+      loadMore={loadMore}
```

Comment:
so this lineup has the "selfLoad" flag enabled which means we shouldnt have to specify the loadMore function. what it sounds like is maybe the fetchLineupMetadatas call internally in the lineup isnt getting all the right data? might be easiest to pair on that.

Reply:
good call out here